packages/cli-client/package.json

```diff
@@ -13,7 +13,10 @@
     "typescript": "2.6.1"
   },
   "description": "Command-line interface for Wire's secure messaging platform.",
-  "files": ["bin", "dist"],
+  "files": [
+    "bin",
+    "dist"
+  ],
```

Comment:
@greysteil this is not breaking the PR.. but we would prefer if it wouldn't modify anything else than the version numbers. I know that it's because of the `yarn add` and I'm not sure how it could be easily fix.. 

We have commit hooks that fixing various formats with Prettier.

Reply:
Interesting. I think we're currently a bit overzealous with updating the formatting of `package.json` files - I'll have a look at moving away from using `yarn add`, which would make it easy for us to honour your existing formatting. Might take a little while.

Reply:
We are not in a rush as this is not really breaking anything.. except our (mostly mine) OCD :)